.github/workflows/run-motion-integration.yml

```
@@ -0,0 +1,113 @@
+    steps:
+      - uses: actions/checkout@v5
+      - name: Patch canonical integration
+        run: |
+          python3 <<'PY'
+          from pathlib import Path
+          root = Path('.')
+          app = root/'app/app.js'; s=app.read_text()
+          if 'case "rise":' not in s:
+            old='''        case "pulse": drawPulseEffect(ctx, w, h, style, lines, time); break;
+        case "typewriter":
+        case "instagram":
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected ',' or '}'

(syntax-check)